evergreen/config.yml

```diff
@@ -654,7 +654,8 @@ tasks:
       script: ../tools/run-tests-on-exfat.sh
 
 - name: valgrind
-  exec_timeout_secs: 18000
+  exec_timeout_secs: 14400
```

Comment:
I assume you checked the actual test run and this is enough

Reply:
The duration for the previous runs (on commit to master) has been just over an hour.